### ISSUE TITLE
Error: Failed to fetch

### ISSUE THREAD

**Issue by Ecco** [bitwarden/browser#1747]
## Describe the Bug

I get an "Error: Failed to fetch" whenever I try to add a new item.

## Steps To Reproduce

1. Try to add a new item
2. There's no step 2
## Additional Context

Running the extension in developer mode gives an explanation why:

```
Access to fetch at 'https://identity.bitwarden.com/connect/token' from origin 'chrome-extension://REDACTED' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

Here's the response to the preflight request:

```
cf-cache-status: DYNAMIC
cf-ray: 63753d341c3057a3-IAD
cf-request-id: 091d24948d000057a39e304000000001
date: Mon, 29 Mar 2021 01:14:02 GMT
expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
server: cloudflare
server-version: 1.40.0
set-cookie: __cfduid=REDACTED; expires=Wed, 28-Apr-21 01:14:02 GMT; path=/; domain=.bitwarden.com; HttpOnly; SameSite=Lax; Secure
strict-transport-security: max-age=31536000; includeSubDomains; preload
x-content-type-options: nosniff
x-rate-limit-limit: 1m
x-rate-limit-remaining: 59
x-rate-limit-reset: 2021-03-29T01:15:02.2833941Z
```

There are indeed no `Access-Control-Allow-` headers in there… 

## Environment

- OS : macOS 11.2.3
- Browser: Chrome 89.0
- Extension : 1.49.1

**Comment by Ecco** [comment]
I deleted the extension and reinstalled it. It worked for like a week. Now the same problem is back again…

**Comment by cscharf** [comment]
note to self for tomorrow, please check cloudflare's access control headers and preflight directives along with service and proxy configurations in host... not sure why this is happening here, but may need to set up a quick macOS 11.2.3 instance VM to test with.

**Comment by Ecco** [comment]
It just started to happen again to me. Let me know if you want me to run any diagnosis!

**Comment by FauxFaux** [comment]
This seems to happen for me (even in the login dialog) if I have "site access" set to "on click", which is not the default.

**Comment by Ecco** [comment]
Oh, yeah, I also have this configuration!

**Comment by cscharf** [comment]
Thanks for that additional detail, much appreciated. Will follow up with our QA team.

**Comment by kendratodd** [comment]
This appears to be a duplicate of https://github.com/bitwarden/browser/issues/1747 and can be consistently reproduced. In favor of consolidating reports, I'm going to close this ticket & cross-reference. Thanks!
